src/main/php/PDepend/Source/Language/PHP/PHPParserVersion80.php

```diff
@@ -185,6 +188,45 @@ protected function parseConstantArgument(ASTConstant $constant, ASTArguments $ar
         return $constant;
     }
 
+    /**
+     * This method parses a function postfix expression. An object of type
+     * {@link ASTFunctionPostfix} represents any valid php
+     * function call.
+     *
+     * This method will delegate the call to another method that returns a
+     * member primary prefix object when the function postfix expression is
+     * followed by an object operator.
+     *
+     * @param  ASTNode $node This node represents the function
+     *        identifier. An identifier can be a static string, a variable, a
+     *        compound variable or any other valid php function identifier.
+     * @return ASTNode
+     * @throws ParserException
+     */
+    protected function parseFunctionPostfix(ASTNode $node)
+    {
+        if ($node instanceof ASTIdentifier && $node->getImage() === 'match') {
+            $image = $this->extractPostfixImage($node);
+
+            $function = $this->builder->buildAstFunctionPostfix($image);
+            $function->addChild($node);
+
+            $this->consumeComments();
+
+            $this->tokenStack->push();
+
+            $function->addChild(
+                $this->parseArgumentsParenthesesContent(
+                    $this->builder->buildAstMatchArgument()
+                )
+            );
+
+            return $function;
+        }
+
+        return parent::parseFunctionPostfix($node);
```

Comment:
Maybe switch the if to quit early and get a smaller indentation block?

```suggestion
        if (!($node instanceof ASTIdentifier) || $node->getImage() !== 'match') {
            return parent::parseFunctionPostfix($node);
        }
        $image = $this->extractPostfixImage($node);

        $function = $this->builder->buildAstFunctionPostfix($image);
        $function->addChild($node);

        $this->consumeComments();

        $this->tokenStack->push();

        $function->addChild(
            $this->parseArgumentsParenthesesContent(
                $this->builder->buildAstMatchArgument()
            )
        );

        return $function;
```